### ISSUE TITLE
Normalized size across monitors (retina)

### ISSUE THREAD

**Issue by AlexisRenchon** [adrianariton/CssMMakieLayout#2]
Hello! 

I am an active user of WGLMakie and JSServe, I wrote a small package, [ParamViz.jl](https://github.com/CliMA/ParamViz.jl), unregistered as well for now, that I use to [deploy web apps on a package docs](https://clima.github.io/ClimaLSM.jl/previews/PR214/dynamicdocs/pages/soil_biogeochemistry/microbial_respiration/). 

An issue I am running into, is that WGLMakie figures are set in pixels, and therefore vary with the resolution of the monitor. On some monitors, my figures appear very big, and others, very small. I wish I could have my figures be a proportion of the monitor or web page... 
It is an issue Simon Danish (author of Makie and JSServe) is aware of, but fixing it is not a priority. 

I imagine css would help, and it seems your package can do just that?
I am not sure though, but I hope your package helps, I will experiment and let you know how it goes!

**Comment by adrianariton** [comment]
Yes, I ran into this problem to and I am to fix it soon in a future version of the library. The problem has to do with the feature of pixel doubling of retina screens, meaning the figures appear 2 times smaller than the intended resolution.

In retina screens each pixels is comprised of a 2x2 grid of smaller "virtual" pixels. The figure resolution is measured in the terms of the small pixels and that's why they appear half the size of what they are supposed to be on a normal screen.

I have come up with something to bypass it:
- I plan on adding a config parameter for retina screens to double the resolution, since the simple scaling with css would mess up the click / selection events inside the canvas. 

Let me know if you find a cleaner way to do this. Any contributions are greatly appreciated!

**Comment by AlexisRenchon** [comment]
Thanks a lot! 
I'm sure Simon would be interested in a solution too.

**Comment by SimonDanisch** [comment]
This will be fixed in the next breaking release, which will hopefully happen soon:
https://github.com/MakieOrg/Makie.jl/pull/3113
